source/plugin/api-versions.rst

```diff
@@ -0,0 +1,81 @@
+============
+API-Versions
+============
+
+This page explains which API versions exists and to which Minecraft version their implementations belong to.
+
++-------------+--------------+----------------+-------------------------------------------+
+| API-Version | Release Date | End of Support | Known Implementations (Minecraft Version) |
++=============+==============+================+===========================================+
+| *Planned*   | ---          | --             | * *SpongeForge (1.13.x)*                  |
+| *8.0.0*     |              |                | * *SpongeVanilla (1.13.x)*                |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.1.0       | 2018-09-06   | Late 2019      | * SpongeForge (1.12.2)                    |
+|             |              |                | * SpongeVanilla (1.12.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.0.0       | 2018-01-01   | ^              | * SpongeForge (1.12.2)                    |
+|             |              |                | * SpongeVanilla (1.12.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 6.0.0       | 2017-05-02   | 2017-12-31     | * SpongeForge (1.11.2)                    |
+|             |              |                | * SpongeVanilla (1.11.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 5.1.0       | 2017-02-25   | 2017-12-31     | * SpongeForge (1.10.2)                    |
+|             |              |                | * SpongeVanilla (1.10.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+
+It is recommended to always update to the latest stable version of the API or to an LTS version.
+
+.. note::
+
+    You can still release your plugin with an older major API version, but we don't officially support these versions.
+
+
+LTS-Versions
```

Comment:
> After adding the End of Support column i'm not sure that we need the LTS explanation anymore.

Yeah, this section seems useless.

Reply:
I hoped I could get a little more info on what Sponge understands as an LTS version aside from getting updates after development of 8.0 has started.

Do we support actually support 7.0.0 in the long term or only its successor 7.1.0?

Reply:
We're only ever going to support the latest API release for a given MC version, so bug-fixes for API 7 are likely to appear in API 7.1. As I understand it, our brand of LTS involves fixing critical bugs and possibly back-porting features from the API 8 branch, if it is sane to do so.

Reply:
'LTS' has never been anything super defined anyway, it's more a term of art to give meaning when we have been talking.

We don't have any defined time frames or policies on what LTS entails, except that 'we plan on supporting it longer / side by side with latest stable'

Reply:
Section removed.